### DISCUSSION TITLE
How do I set the baudrate of an endpoint?

### DISCUSSION THREAD

**Discussion by Jean-Lytehouse** [pyusb/pyusb#466] [Q&A] [answered]
Pretty much the title - I need to set the baudrate of the OUT endpoint to 2400 as that is what my device is expecting - how can I do this?

**Comment by mcuee** [comment, marked as answer]
There is no such thing as Baudrate for an OUT endpoint. USB is different from serial.

If you want to set the baud rate of an USB to Serial Converter, try pyserial. pyusb is not the right library.
